zeek/compiler/glue-compiler.cc

```diff
@@ -370,7 +373,7 @@ bool GlueCompiler::loadEvtFile(std::filesystem::path& path) {
             _locations.pop_back();
         }
 
-    } catch ( const ParseError& e ) {
+    } catch (const ParseError& e) {
```

Comment:
Looks like it picked up the wrong clang-format here. I'll fix that before merge.